game/debug/view/resource_list.lua

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Is this script too different from `debug/view/domain_list.lua`? Can we not use the same code?

Reply:
I thought the same thing but I think I forgot to refactor it. I’ll do it, it shouldn’t be too hard